docs/Pitfalls.asciidoc

```diff
@@ -53,3 +53,25 @@ impacted by snapshot creation. Loading VM snapshots can also have an impact on
 SUT behavior as the execution of the first step after loading a snapshot might
 be delayed. This can lead to problems if the executed tests do not foresee an
 appropriate timeout margin.
+
+== DB migration from sqlite to postgreSQL
+As a first step  to start using postgreSQ, please, configure postgreSQL database
```

Comment:
One whitespace too much and postgreSQ is missing an l. Besides, I would prefer spelling the databases "PostgreSQL" and "SQLite" (mind the case).